README.md

```diff
@@ -51,12 +55,15 @@ Simply copy the **menas.war** file produced when building the project into Tomca
 #### Standardization and Conformance requirements:
 - **Spark 2.4.0** installation
 - **Hadoop 2.7** installation
+- **Menas** running instance
 - **Menas Credentials File** in your home directory (a configuration file for authenticating the Spark jobs with Menas) 
 e.g. `~/menas-credential.properties`:
 ```
 username=user
 password=changeme
 ```
+ - **_INFO file** must be present in the **RAW** dataset directory (this is a file tracking control measures via [Atum](https://github.com/AbsaOSS/atum), example can be found [here](examples/data/input/_INFO))
```

Comment:
`in the **RAW** next to the actual data`. So they don't put it into <path_to_dataset_in_menas>

Reply:
I switched the order of the `dir structure` and `_INFO file` to give it the right context and avoid confusion like this.